### PR TITLE
fix: increase in modal width by 3% to resolve issue #1786

### PR DESCRIPTION
**Pull request by sahell0x**
## Description
issue : #1786 

how it is working now:
[Screencast from 2024-10-25 16-42-48.webm](https://github.com/user-attachments/assets/f12ab9cb-8416-4066-865f-27544724ddda)

## What type of PR is this? (check all applicable)

- [ ] 🍕 Feature
- [x] 🐛 Bug Fix
- [ ] 📝 Documentation Update
- [x] 🎨 Style
- [ ] 🧑‍💻 Code Refactor
- [ ] 🔥 Performance Improvements
- [ ] ✅ Test
- [ ] 🤖 Build

## Have you tested locally?

- [x] 👍 yes
- [ ] 🙅 no, because I am lazy

## Added tests?

- [ ] 👍 yes
- [x] 🙅 no, because they aren't needed
- [ ] 🙋 no, because I need help

## Added to documentation?

- [ ] 📜 README.md

### CODE REVIEW

**Review by itsmdasifraza — APPROVED** [review]

File: packages/@sparrow-workspaces/src/hoc/WithModal.svelte [hunk 1]
```diff
@@ -15,7 +15,7 @@
 <Modal
   title={"Save Changes?"}
   type={"dark"}
-  width={"35%"}
+  width={"500px"}
```
Comment (itsmdasifraza) [inline packages/@sparrow-workspaces/src/hoc/WithModal.svelte:18]: can we do it in percentage.

Comment (sahell0x) [reply packages/@sparrow-workspaces/src/hoc/WithModal.svelte:18]: I checked the percentage approach, but 40% modal looks little bit wide, and at 38%, the margins do not align well. I also looked at similar applications, and they seem to follow a standard pattern. If a 40% width works with the requirements and we’re okay with a wider look in full screen, we can go with it. Let me know what you think !!

Screenshot from 2024-10-28 18-43-57
Screenshot from 2024-10-28 18-44-43

File: apps/@sparrow-desktop/src-tauri/gen/schemas/linux-schema.json [hunk 2]
```diff
@@ -0,0 +1,2402 @@
...
+        {
+          "$ref": "#/definitions/Capability"
+        }
+      ]
+    },
+    {
+      "description": "A list of capabilities.",
+      "type": "object",
+      "required": [
+        "capabilities"
```
Comment (itsmdasifraza) [inline apps/@sparrow-desktop/src-tauri/gen/schemas/linux-schema.json:18]: Is this required?

File: packages/@sparrow-workspaces/src/hoc/WithModal.svelte (hunk 1 above)
Comment (itsmdasifraza) [reply packages/@sparrow-workspaces/src/hoc/WithModal.svelte]: Will enhance it further, Looks good to me for now.
Good work 🎇

**Pull request merged**